### PR TITLE
INHEART-148: Add AuctionCard

### PR DESCRIPTION
**Pull request by pmusialik-intive**
image

Added ImageBackground atom
image

Initialized single auction page just to check if the redirection from auction card works (on title press)
Changed order in auction hearts since we always want to have icon in one place (heartcoins <-> heart icon)

### CODE REVIEW

**Comment by pawelroguski** [comment]
Please resolve conflict first ;)

**Comment by KonradKarlinski** [comment]
Please add properly description and more screens because I notice in code u modified and added more then only auction cart. 

- [ ] 正しい説明文とスクリーンショットを追加

File: libs/ui/molecules/src/lib/AuctionHearts/AuctionHearts.tsx [hunk 1]
```diff
@@ -1,22 +1,17 @@
...
+  const { quantity } = props;
 
   return (
     <Row space={2} alignItems='center'>
-      <Icon name={IconType.favoriteOutline} size={35} color={colors.secondary[600]} />
-      <TextStyled fontSize='xl' testID={testID}>
-        {quantity}
-      </TextStyled>
+      <Text>{quantity}</Text>
+      <Icon name={IconType.favoriteOutline} size={30} color={colors.secondary[600]} />
```
Comment (KonradKarlinski) [inline libs/ui/molecules/src/lib/AuctionHearts/AuctionHearts.tsx:14]: Why did you delete testID ?

- [ ] Explain

Comment (pmusialik-intive) [reply libs/ui/molecules/src/lib/AuctionHearts/AuctionHearts.tsx:14]: 1. There was no added value in test since we're checking if component renders quantity
2. Prop title was misleading because it suggests that it is AuctionHearts test id - but it was somewhere deeper - assigned to a text

**Review by KonradKarlinski — APPROVED** [review]
